src/misc/base64/base64_decode.c

```diff
@@ -156,6 +171,20 @@ int base64_strict_decode(const char *in,  unsigned long inlen,
 {
    return _base64_decode_internal(in, inlen, out, outlen, map_base64, strict);
 }
+
+/**
+   Relaxed base64 decode a block of memory
```

Comment:
Probably this (and the URL variant) comment should be changed?

Reply:
done